invenio_drafts_resources/links.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-Records-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Invenio Resources module to create REST APIs."""
+
+
+from invenio_records_resources.links import RecordLinkBuilder, api_route
+
+
+class DraftSelfLinkBuilder(RecordLinkBuilder):
+    """Builds draft self link."""
+
+    def __init__(self, config):
+        """Constructor."""
+        super(DraftSelfLinkBuilder, self).__init__(
+            key="self",
+            route=api_route(config.draft_route),
+            action="read",
+            permission_policy=config.permission_policy_cls
+        )
+
+
+class DraftSelfHtmlLinkBuilder(RecordLinkBuilder):
+    """Builds draft "self_html" link."""
+
+    def __init__(self, config):
+        """Constructor."""
+        super(DraftSelfHtmlLinkBuilder, self).__init__(
+            key="self_html",
+            # TODO: Invenio-App-RDM needs to set deposit page in config
+            route="/deposits/<pid_value>/edit",
```

Comment:
Yes. Making an issue for it: https://github.com/inveniosoftware/invenio-app-rdm/issues/232 and will deal with it separately, since it affects at least invenio-app-rdm and react-invenio-deposit too.